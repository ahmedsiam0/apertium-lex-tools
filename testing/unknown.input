^potato<n>/known<n>/unknown<n>$ ^normal_word<adj>/normal_word<adj>$
^potato<n>/known<n>/unknown<n>$ ^*unknown_word/*unknown_word$
