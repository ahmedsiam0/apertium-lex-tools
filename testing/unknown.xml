<rules>
  <rule>
	  <match lemma="potato"><select lemma="known"/></match>
  </rule>
  <rule weight="3">
    <match lemma="potato">
      <select lemma="unknown"/>
    </match>
    <match tags=""/>
  </rule>
</rules>
