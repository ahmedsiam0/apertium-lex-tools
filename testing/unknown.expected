^potato<n>/known<n>$ ^normal_word<adj>/normal_word<adj>$
^potato<n>/unknown<n>$ ^*unknown_word/*unknown_word$
